src/messages.ts

```diff
@@ -70,9 +70,45 @@ export const DEFAULT_PROJECT_ID_DEPRECATED = {
   ].join(' ')
 };
 
-export const REFRESH_ACCESS_TOKEN_DEPRECATED = {
+export const COMPUTE_CREATE_SCOPED_DEPRECATED = {
   code: 'google-auth-library:DEP003',
   type: WarningTypes.DEPRECATION,
+  message: [
+    'The `createScopedRequired` method on the `Compute` class has been deprecated,',
+    'and will be removed in the 3.0 release of google-auth-library.'
+  ].join(' ')
+};
+
+export const JWT_CREATE_SCOPED_DEPRECATED = {
```

Comment:
nit: perhaps a single deprecation was sufficient. `createScopeRequired` has been deprecated on _all_ classes.

Reply:
Ah - let me explain!  The rationale is that we only warn once for each deprecation.  I wanted to make sure that we warn once for each separate API the user is off and using.  I could see this one both ways, so I err'd on the side of over-communicating.

Reply:
👍